controllers/eventsIndexController.js

```diff
@@ -1,26 +1,20 @@
-const db = require('../db');
+const db = require("../db");
 
 exports.getEventsIndex = async (req, res) => {
-    console.log("GET: Hello getEvents")
-    try { 
-        const collectionEvents = db.getDb().db('softball').collection('events')
-        const allEvents = await collectionEvents.find(
-            {},
-            {_id: 0, date: 1, result: 1}
-        )
-        .sort({ date: -1 })
-        .toArray()
-        console.log('allEvents', allEvents)
-        res
-          .status(200)
-          .json(allEvents);
-    } catch(err) {
-        console.log(err)
-        res
-          .status(500)
-          .json({
-            error: 'db-events-could-not-find',
-            message: err
-        });
-    }
-};
\ No newline at end of file
+  console.log("GET: Hello getEvents");
+  const SortedOrder = (req.query.sort = "desc" ? -1 : 1);
```

Comment:
If `sort` is not give on the request, it would go `1`, which is "asc". 
But we want to have a default behavior with `desc`. "Desc" generally is used more so it would be good idea to have a default as "desc".

Reply:
so you mean, `const sortedOrder = (req.query.sort = "desc" ? -1 : -1);`?

Reply:
I mean this `sortedOrder` should be `-1` when `req.query.sort` is `'desc'` or `undefined and `1` when it's `"asc"`.

Reply:
try to test the endpoint with curl or Postman, with the query param and without it to test the default behaviour.